### PR TITLE
CB-1468 Add logging to Environment Service

### PR DESCRIPTION
**Pull request by topolyai5**

### CODE REVIEW

File: environment/src/main/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <appender name="FILE-${nodeid}" class="ch.qos.logback.core.rolling.RollingFileAppender">
+                <file>cloudbreak-log/cloudbreak-${nodeid}.log</file>
+                <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+                    <layout class="com.sequenceiq.cloudbreak.logger.MaskingPatternLayout">
+                        <loggerNameFilter>com.sequenceiq</loggerNameFilter>
+                        <pattern>%date{ISO8601} [%thread] %method:%line %-5level %logger{5} - [instance:${nodeid}] [type:%X{resourceType:-springLog}] [id:%X{resourceId:-}] [name:%X{resourceName:-}] [requestid:%X{requestId:-}] [tenant:%X{tenant:-}] [workspaceId:%X{workspaceId:-}] [userId:%X{userId:-}] [userName:%X{userName:-}] %msg%n</pattern>
+                    </layout>
+                </encoder>
+                <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+                    <FileNamePattern>cloudbreak-log/cloudbreak-${nodeid}-%d{yyyy-MM-dd}.log</FileNamePattern>
```
Comment (doktoric) [inline environment/src/main/resources/logback.xml]: do we need this?

File: environment/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    <logger name="org.springframework.boot.autoconfigure.security.AuthenticationManagerConfiguration" level="ERROR"/>
+    <logger name="org.springframework.statemachine.support" level="ERROR"/>
+    <logger name="com.sequenceiq.ambari.client.AmbariClientUtils" level="ERROR"/>
+    <logger name="org.apache.kafka.clients.producer" level="ERROR"/>
+    <logger name="com.amazonaws.request" level="DEBUG"/>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="com.sequenceiq.cloudbreak.logger.MaskingPatternLayout">
```
Comment (doktoric) [inline environment/src/main/resources/logback.xml]: is this class available ?

File: environment/src/main/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        </sift>
+    </appender>
+
+    <root level="${CB_ROOT_LOG_LEVEL:-INFO}">
+        <appender-ref ref="CB_NODEID_BASED"/>
+        <appender-ref ref="STDOUT"/>
+    </root>
+
+    <logger name="com.sequenceiq" level="${CB_LOG_LEVEL:-DEBUG}" additivity="false">
+        <appender-ref ref="CB_NODEID_BASED"/>
```
Comment (doktoric) [inline environment/src/main/resources/logback.xml]: CB_NODEID_BASED is probably not the best name

File: environment/src/main/resources/logback.xml (hunk 1 above)
Comment (topolyai5) [reply environment/src/main/resources/logback.xml]: I think, yes, we do. Just I've renamed to environment-log

File: environment-log/environment-default.log [hunk 4]
```diff
@@ -0,0 +1,1867 @@
+2019-05-21 10:12:27,698 [main] logStarting:50 INFO  c.s.e.EnvironmentApplication - [instance:default] [type:springLog] [id:] [name:] [requestid:] [tenant:] [userId:] [userName:] Starting EnvironmentApplication on HW12513.local with PID 36046 (/Users/tgergely/projects/cloudbreak/environment/out/production/classes started by tgergely in /Users/tgergely/projects/cloudbreak)
```
Comment (attilapalfi92) [inline environment-log/environment-default.log]: please remove this log file

**Pull request merged**